tools/tests/test_osqueryd.py

```diff
@@ -45,15 +45,18 @@ def test_2_daemon_with_option(self):
                 "logger_path": logger_path,
                 "verbose": True,
             })
-        info_path = test_base.getLatestInfoLog(logger_path)
 
         self.assertTrue(daemon.isAlive())
 
         def info_exists():
+            info_path = test_base.getLatestInfoLog(logger_path)
```

Comment:
Good call. I'll extend the test to check for readable. I think we're aren't as concerned if the integration is able to write though? If the integration can open and read, and the read is > 0 I think that'd suffice right?